server/controllers/stock/reports/stock_movements.report.handlebars

```diff
@@ -76,7 +76,7 @@
                   <td>{{label}}</td>
                   <td class="text-right">{{quantity}}</td>
                   <td class="text-right">{{unit_cost}}</td>
-                  <td class="text-right">{{multiply quantity unit_cost}}</td>
+                  <td class="text-right">{{currency cost ../metadata.enterprise.currency_id}}</td>
```

Comment:
I'm almost certain this should be `../../metadata.enterprise.currency_id`.  Have you tried changing the enterprise currency to see if this changes?

Reply:
So, it looks like I was wrong - I tested this, and the currency value does not change.

If I'm wrong in the future, please tell me instead of just blindly changing the code.